.devcontainer/postCreateCommand.sh

```diff
@@ -1,5 +1,10 @@
 #!/bin/bash
 
+# move script rstudio into /usr/bin
+cp "$(pwd)/.devcontainer/rstudio.sh" /usr/bin/rstudio
```

Comment:
We should also add `sudo` to cp and `chmod +x` to `/usr/bin/rstudio`

```suggestion
sudo cp "$(pwd)/.devcontainer/rstudio.sh" /usr/bin/rstudio
sudo chmod +x /usr/bin/rstudio
```